feeadjuster/feeadjuster.py

```diff
@@ -243,6 +262,12 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
     "Deactivate update threshold randomization and hysterisis.",
     "flag"
 )
+plugin.add_option(
+    "feeadjuster-deactivate-event-subscription",
+    False,
+    "Deactivate forward_event subscription.",
```

Comment:
What about

> Deactivate new fee updates

This seems more user-friendly to describe the behaviour rather than the technical solution